mitodl/ocw-hugo-themes
# metadata tags added for sharing on Facebook

#### Pre-Flight checklist


#### What are the relevant tickets?
- Closes https://github.com/mitodl/ocw-hugo-themes/issues/464

#### What's this PR do?
- Adds metadata tags for sharing on facebook

#### How should this be manually tested?
- Copy the URL of course deployed on netlify on this PR. (either [course](https://ocw-hugo-themes-course-pr-823--ocw-next.netlify.app/) or [course-v2](https://ocw-hugo-themes-course-v2-pr-823--ocw-next.netlify.app/))
- Test the page on [Facebook sharing debugger](https://developers.facebook.com/tools/debug/)
- Repeat the above steps with multiple/different course pages
- OR: 
- Checkout this branch
- Build any course
- View page source
- Verify that all the Facebook Open Graph tags are present in the head with correct values/content.   

#### Screenshots (if appropriate)

Shared on Facebook (view mode is `only me` so this is not visible to public)

<img width="602" alt="image" src="https://user-images.githubusercontent.com/93309234/184631743-7e94e727-e499-40f0-8a0d-0564035a716d.png">

<img width="600" alt="image" src="https://user-images.githubusercontent.com/93309234/184631942-635d1867-945a-40d2-9b39-9389a3825d5b.png">
